test/enable-pathToRegexpModule.test.js

```
@@ -0,0 +1,135 @@
+  let host;
+  before(() => {
+    app = mock.app({
+      baseDir: 'apps/fileModeMatch-glob-with-pathToRegexpModule',
+      pathToRegexpModule: require.resolve('path-to-regexp-v8'),
+    });
+    return app.ready();
+  });
+  before(() => {
+    server = app.listen();
+    host = 'http://127.0.0.1:' + server.address().port;
+  });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow before duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)